### DISCUSSION TITLE
Is VN1670 supported?

### DISCUSSION THREAD

**Discussion by is02ljie** [hardbyte/python-can#1577] [Q&A] [answered]
I have some VN1670 devices, but I can not find 1670 support in xldefine.py. 
```
class XL_HardwareType(IntEnum):
    XL_HWTYPE_NONE = 0
    XL_HWTYPE_VIRTUAL = 1
   ...
    XL_HWTYPE_VN1630 = 57
    XL_HWTYPE_VN1640 = 59
    XL_HWTYPE_VN8970 = 61
    XL_HWTYPE_VN1611 = 63
}
```

Is XL driver library support VN1670?

**Comment by zariiii9003** [comment, marked as answer]
VN1670 would be value 120. It is not available in the XL_HardwareType Enum yet, but you should be able to use python-can anyway without any issues.
...unless the vector vxlapi64.dll does not support it yet 🤷‍♂️
